rss-client/src/app/services/user.service.ts

```diff
@@ -33,11 +33,6 @@ export class UserService {
     headers: new HttpHeaders({ 'Content-Type': 'application/json' }),
   };
 
```

Comment:
Why is anything in here being changed?

Reply:
Again, I was under the impression that this was mock data that my team added and not yours. If these changes are unnecessary then 100% discard them